scripts/check-branch.sh

```diff
@@ -7,7 +7,7 @@ email="${4:-missing}"
 token="${5:-missing}"
 
 
-exists=$(gh api -X GET "repos/${repo}/branches" --jq '.[].name | select(. == "${branch}")')
+exists=$(gh api -X GET "repos/${repo}/branches" --jq ".[].name | select(. == \"${branch}\")")
```

Comment:
this was an error with jq/bash. Bash couldn't expand `${branch}` because it was in single quotes and apparently JQ doesn't like single quotes either.